presto-server-rpm/pom.xml

```diff
@@ -33,143 +35,159 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
This bit isn't changed except updating the version of redlinerpm-maven-plugin to 2.0.  It just used to be in it's own profile and I pulled it out.